jetson/percep/pcl.cpp

```diff
@@ -102,7 +102,7 @@ void CPUEuclidianClusterExtraction(pcl::PointCloud<pcl::PointXYZRGB>::Ptr & pt_c
     pcl::EuclideanClusterExtraction<pcl::PointXYZRGB> ec;
     ec.setClusterTolerance (60); // 60 mm radius per point
     ec.setMinClusterSize (20);
-    ec.setMaxClusterSize (10000);
+    ec.setMaxClusterSize (100000);
```

Comment:
NIT: I'd like to see this end up in a config file

Reply:
Also noted